crates/astria-sequencer/src/app/tests_breaking_changes.rs

```diff
@@ -238,5 +241,29 @@ async fn app_execute_transaction_with_every_action_snapshot() {
 
     let signed_tx = tx.into_signed(&alice_signing_key);
     app.execute_transaction(signed_tx).await.unwrap();
+
+    // execute BridgeUnlock action
+    let tx = UnsignedTransaction {
+        params: TransactionParams {
+            nonce: 0,
+            chain_id: "test".to_string(),
+        },
+        actions: vec![
+            BridgeUnlockAction {
```

Comment:
you can put this action into the big transaction above, just put it under the `BridgeLockAction` so that the `BridgeLockAction` is executed first